lib/config/config-validator.js

```diff
@@ -97,7 +101,10 @@ function validateRuleOptions(id, options, source) {
 
         if (!validSeverity) {
             message.push(
-                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed \"", severity, "\").\n");
+                "\tSeverity should be one of the following: 0 = off, 1 = warning, 2 = error (you passed `",
+                util.inspect(severity).replace(/'/g, "\"").replace(/\n/g, ""),
```

Comment:
And we use single quotes for values in all of our messages, so this is consistent.